rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -448,36 +448,40 @@ def parse_message_file(pkg_name, interface_filename):
             pkg_name, msg_name, h.read())
 
 
+def extract_file_level_comments(message_string):
+    lines = message_string.splitlines()
+    index = next(
+        (i for i, v in enumerate(lines) if not v.startswith(COMMENT_DELIMITER)), -1)
+    if index != -1:
+        file_level_comments = lines[0:index]
+        other = lines[index:]
+    else:
+        file_level_comments = []
+        other = lines[:]
+    file_level_comments = [line.lstrip(COMMENT_DELIMITER) for line in file_level_comments]
+    return file_level_comments, other
+
+
 def parse_message_string(pkg_name, msg_name, message_string):
-    file_level_ended = False
     message_comments = []
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/ros2/rosidl/pull/632/commits/5b42eb0a7041feea4e4be6e0ed5f5d0b56c17523.